### PR TITLE
Add power shell example

### PR DESCRIPTION
**Pull request by friism**
**\- What I did**

The example command doesn't work on Windows, so added Powershell example

**\- How to verify it**

Build without context on Windows system.

### CODE REVIEW

File: docs/reference/commandline/build.md [hunk 1]
```diff
@@ -79,7 +79,11 @@ Build Syntax Suffix | Commit Used | Build Context Used
 Instead of specifying a context, you can pass a single Dockerfile in the `URL`
 or pipe the file in via `STDIN`. To pipe a Dockerfile from `STDIN`:
 
-    docker build - < Dockerfile
+    $ docker build - < Dockerfile
+
+With Powershell on Windows, you can run:
+
+    Get-Content Dockerfile | docker build -
```
Comment (SvenDowideit) [inline docs/reference/commandline/build.md]: I'm guessing the `cmd.exe` version looks like

`type Dockerfile | docker build -` ?

**Comment by SvenDowideit** [comment]
@friism can you squash your commits?

otherwise, LGTM

**Comment by thaJeztah** [comment]
LGTM after commits are squashed

File: docs/reference/commandline/build.md (hunk 1 above)
Comment (friism) [inline docs/reference/commandline/build.md]: Haven't tested. It's worth debating whether we should have examples in `cmd.exe` too. I don't know what's the usage of different shells on Windows, but I somehow hope most people are on Powershell (if they're not using a Windows Bash implementation)

**Comment by friism** [comment]
thx, rebased

**Comment by thaJeztah** [comment]
thanks!

**Pull request merged**